applications/openshift/api-server/api_server_request_timeout/rule.yml

```diff
@@ -2,32 +2,30 @@ documentation_complete: true
 
 prodtype: ocp4
 
-title: 'Configure the API Server Request Timeout'
+title: 'Configure the API Server Minimum Request Timeout'
 
 description: |-
-    All components that use the API should connect via the secured port,
-    authenticate themselves, and be authorized to use the API. To ensure
-    such a configuration, edit the <tt>openshift-kube-apiserver</tt> configmap
-    and set the <tt>min-request-timeout</tt> to <tt>300</tt>:
+    The API server minimum request timeout defines the minimum number of
```

Comment:
Could you also add to this description a link to the docs on how to fix this if folks need to change this value? e.g. through the `KubeletConfig` object: https://docs.openshift.com/container-platform/4.6/nodes/nodes/nodes-nodes-managing.html